src/api-engine/api/routes/channel/views.py

```diff
@@ -0,0 +1,150 @@
+from rest_framework import viewsets, status
+from rest_framework.response import Response
+from rest_framework_jwt.authentication import JSONWebTokenAuthentication
+
+from drf_yasg.utils import swagger_auto_schema
+
+from django.core.exceptions import ObjectDoesNotExist
+from django.core.paginator import Paginator
+
+from api.common.serializers import PageQuerySerializer
+from api.utils.common import with_common_response
+from api.auth import TokenAuth
+from api.exceptions import (
+    ResourceNotFound,
+)
+from api.models import (
+    Organization,
+    Channel
+)
+from api.routes.channel.serializers import (
+    ChannelCreateBody,
+    ChannelIDSerializer,
+    ChannelListResponse,
+    ChannelResponseSerializer,
+    ChannelUpdateSerializer
+)
+
+
+class ChannelViewSet(viewsets.ViewSet):
+    """Class represents Channel related operations."""
+    authentication_classes = (JSONWebTokenAuthentication, TokenAuth)
+
+    @swagger_auto_schema(
+        query_serializer=PageQuerySerializer,
+        responses=with_common_response(
+            {status.HTTP_201_CREATED: ChannelListResponse}
+        ),
+    )
+    def list(self, request):
```

Comment:
If you want to filter out the channel of the organization, I think it will be more convenient for you to do so. channel = organization.channels
https://github.com/hyperledger/cello/blob/96d9cb4b7ad10498daf2aedb76b0062859366048/src/api-engine/api/models.py#L89